indra/sources/trips/client.py

```diff
@@ -123,9 +123,9 @@ def save_xml(xml_str, file_name, pretty=True):
                                       ['string=', 'file=', 'output=', 'help'])
     for o, p in opts:
         if o in ['-h', '--help']:
-            print('String mode: python trips_client.py '
+            print('String mode: python client.py '
                   '--string "RAS binds GTP" --output text.xml')
-            print('File mode: python trips_client.py --file test.txt '
+            print('File mode: python client.py --file test.txt '
```

Comment:
Perhaps this message should be refactored for clarity? e.g. `python trips/client.py ....`, or maybe even `python -m indra.sources.trips.client ...`?

Reply:
Changed to what you suggested "-m indra.sources.trips.client"

Reply:
I edited that here in github